src/community.py

```diff
@@ -37,13 +37,11 @@ def post(self):
 		return hsh_to_return
 
 class AddToCommunity(Resource):
-	def post(self):
+	def post(self,user_id):
```

Comment:
When adding a user to a community, the ID of the user should be sent in the request body, so the `user_id` parameter can be removed. I think the endpoint for adding a user to a community would look something like `/communities/:community_id/users/`

Reply:
@aaron2212  Is it safe to include community's API-KEY in the URL ?